cypress/integration/Layer/stacking/index.js

```diff
@@ -34,19 +34,35 @@ Given(
 )
 
 Then('the second layer is on top of the first layer', () => {
-    assertLayerIsOnTop('second')
+    cy.get('body').click()
+    cy.window().should(win => {
+        expect(win.onLayerClick).to.be.calledOnce
+        expect(win.onLayerClick).to.be.calledWith('second')
+    })
```

Comment:
This wants to be refactored hehe

```js
cy.window()
  .its('onLayerClick')
  .should('be.calledOnce')
  .and('be.calledWith', 'second')
```

Reply:
Yeah that seems nice. I think that that could be filed under point 1. here: https://github.com/dhis2/ui/issues/103#issuecomment-633571972. I'll address it when fixing that issue if you don't mind.